transforms/no-vars.js

```diff
@@ -0,0 +1,79 @@
+/** 
+ * Transform all your `var`s to `let` and `const`.
+ *
+ * The only catch is it may not catch nested bodies that mutate anything beyond
```

Comment:
I don't think we have something built into jscodeshift or recast to do this right now, but I might be mistaken. I did this manually a while ago: https://github.com/cpojer/js-codemod/blob/master/transforms/arrow-function-arguments.js#L17

cc @fkling @benjamn